### PR TITLE
Update versions for 7.0.0 RTM

### PR DESCRIPTION
**Pull request by dougbu**

### CODE REVIEW

**Review by smitpatel — APPROVED** [review]

File: eng/Versions.props [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
-    <PreReleaseVersionLabel>rc</PreReleaseVersionLabel>
-    <PreReleaseVersionIteration>2</PreReleaseVersionIteration>
+    <PreReleaseVersionLabel>rtm</PreReleaseVersionLabel>
     <IncludeSourceRevisionInInformationalVersion>False</IncludeSourceRevisionInInformationalVersion>
     <IsServicingBuild Condition="'$(PreReleaseVersionLabel)' == 'servicing'">true</IsServicingBuild>
     <!--
         When StabilizePackageVersion is set to 'true', this branch will produce stable outputs for 'Shipping' packages
     -->
-    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
+    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">true</StabilizePackageVersion>
```
Comment (dougbu) [inline eng/Versions.props:10]:
```suggestion
    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
```

**Comment by bricelam** [comment]
Is `StabilizePackageVersion=true` blocked on our SQLitePCLRaw dependency or was it reverted for some other reason?

**Comment by dougbu** [comment]
> or was it reverted for some other reason?

This. It's too early to stabilize versions, partly because that makes daily builds hard to consume and partly because the rest of the stack isn't stabilizing yet.

**Pull request merged**